apps/tasks/lib/screens/landing_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/theme.dart';
+import 'package:provider/provider.dart';
+
+/// The Landing Screen of the Application
+class LandingScreen extends StatelessWidget {
+  const LandingScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      floatingActionButton: Padding(
+        padding:
+            EdgeInsets.only(bottom: MediaQuery.of(context).size.height * 0.04),
```

Comment:
Is it possible to add more padding within this button?
The padding for `left = right` should be wider than `top = bottom`...